packages/webchat/test/app.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react'
+import ReactDOM from 'react-dom'
+import { ExposedWebChat, Config } from '../src/index'
+
+export const Webchat: React.FC = () => {
+  const [config, setConfig] = useState<Config>({
+    messagingUrl: 'http://localhost:3100',
+    clientId: ''
+  })
+
+  const search = window.location.search
+
+  useEffect(() => {
+    const config = new URLSearchParams(search).get('config')
+
+    if (!config) {
+      return
+    }
+
+    setConfig(JSON.parse(config))
+  }, [search])
```

Comment:
Watching changes on the query params allows us to update the webchat config during the test so that we can easily test different configs.